lib/v2/darwinawards/index.js

```diff
@@ -0,0 +1,56 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://darwinawards.com';
+    const currentUrl = `${rootUrl}/darwin`;
```

Comment:
```suggestion
    const currentUrl = `${rootUrl}/darwin/`;
```
`/darwin` will be redirected to `/darwin/`

Reply:
🤝